src/test/routes/cylinder-set/updateCylinderSet.test.ts

```diff
@@ -0,0 +1,126 @@
+import { describe, test, expect, beforeAll, afterAll } from '@jest/globals';
+import { FastifyInstance } from 'fastify';
+import { createTestDatabase, dropTestDatabase } from '../../../lib/testUtils';
+import { knexController } from '../../../database/database';
+import { buildServer } from '../../../server';
+
+describe('update cylinder set', () => {
+  const getTestInstance = async (): Promise<FastifyInstance> =>
+    buildServer({
+      routePrefix: 'api',
+    });
+
+  beforeAll(async () => {
+    await createTestDatabase('update_cylinder_set');
+  });
+
+  afterAll(async () => {
+    await dropTestDatabase();
+    await knexController.destroy();
+  });
+
+  test('it responds with 200 and proper body if update was successful; no cylinders updated', async () => {
```

Comment:
Nää testicaset voisi jaoitella positiivisiin ja negatiivisiin caseihin:
`test.describe('joku kuvaava positiivinen teksti, async () => {
  test('it responds with 200 and proper body if update was successful; no cylinders updated', ....)
)`

Reply:
Nojuu ei huono idea

Reply:
![image](https://user-images.githubusercontent.com/45848534/196682668-bbbdbe92-a9e6-4e8d-8a60-a339ccee672a.png)